src/types/shared.ts

```diff
@@ -148,6 +162,7 @@ export type FlightsConditions = {
  * passengers if you require this level of granularity.
  */
 export type OfferSliceConditions = FlightsConditions & {
+  // TODO: ask James about this
```

Comment:
I left this comment here yesterday but after reading it again, I've answered my own question (I thought refund conditions would not be here, but they are, so all good). Safe to remove!

 ```suggestion
```